security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc

```diff
@@ -0,0 +1,21 @@
+[id="using-oc-compliance-plug-in"]
+= Using the oc Compliance Plug-in
+include::modules/common-attributes.adoc[]
+:context: oc-compliance-plug-in-understanding
+
+toc::[]
+
+Although the xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Compliance Operator] automates many of the checks and remediations for the cluster, the full process of bringing a cluster into compliance often requires administrator interaction with the Compliance Operator API and other components. The `oc` Compliance Plug-in makes the process easier.
```

Comment:
I think, the `oc compliance` should be combine. The `oc compliance` Plug-in makes the process easier

Reply:
@ahardin-rh Hi, I have gone through the updated changes and those looks good to me. Here, I am expecting 'oc-compliance' instead of `oc compliance`